### DISCUSSION TITLE
Compare to Mocks Server

### DISCUSSION THREAD

**Discussion by monolithed** [mswjs/msw#1890] [General]
I would like someone to add a comparison with [Service Mocker](https://www.mocks-server.org). It has a very simple declarative API. When I was choosing a solution for working with mock objects, I stopped my choice on it because I didn't even need to read the documentation to figure out how to use it. The onboarding process took less than a minute. It's a matter of personal preference, but I personally really like how they decomposed the API calls for variant collections and supported OpenAPI.

**Comment by kettanaito** [comment]
Hey, @monolithed. 

Here are my initial thoughts after browsing through the Mocks Server docs (not sure why you mention "Service Mocker"—that's a different library):

1. Overall, Mocks Server has the same set of downsides as JSON Server, with which we do have a [comparison](https://mswjs.io/docs/comparison#json-server): having to spawn a designated server, route all your traffic to it (thus deviating it from what happens on prod), using abstract API to define routes, etc. 
2. No GraphQL support.
3. Mocks Server takes a somewhat compound approach to mocking (routes are here, collections are here, here's an abstract API that binds them). You can use all that setup with MSW if you like it but you _don't have to_. 

Those are the main differences. 

As much as I'd like to feature comparisons with a lot more libraries out there, I find it to be counter-productive to the reader of our docs. Even existing comparisons can apply to a lot of tools you find in the wild because approaches to API mocking aren't that drastically different in their core.
